conf/drillTestConfig.properties.example

```diff
@@ -41,7 +41,21 @@ DRILL_TESTDATA=/drill/testdata
 ########################################
 ZOOKEEPERS=IPADDR1:5181/drill/drillbits1
 
+########################################
+# DRILL JDBC driver
+########################################
+#JDBC_DRIVER=com.mapr.drill.jdbc41.Driver
```

Comment:
Please add comments indicating which lines should be used for the Apache driver and which lines should be used for the Simba driver.